subworkflows/nf-core/utils_nfcore_pipeline/tests/main.function.nf.test

```diff
@@ -21,6 +21,26 @@ nextflow_function {
         }
     }
 
+    test("Test Function checkProfileProvided") {
```

Comment:
How about a test with a faulty input?

Reply:
I couldn't find a way to provide a `workflow.profile` in the nf-test docs, nor how to check the content of the warning logging, so I'm not sure how to test a case of failure in this case.

Reply:
Okay, maybe I am thinking too much in a pytest way. 🙂 Good as it is then